### PR TITLE
DPE-196: confirmation-page

### PR DESCRIPTION
**Pull request by lewis-coulson-dit**
Adds the confirmation page for the new "Digital point of Entry" contact support form. Also addresses design feedback from chad across all steps.

Screenshot 2023-06-26 at 13 47 02

To test locally go to "http://greatcms.trade.great:8020/contact/domestic/export-support/"

Fill in all steps of form and then click "Continue" button. You will be redirected to the confirmation page after the review/check your answers page.

On successful submit the form redirects to an empty page where the seventh step form will be located in a future ticket.

### Workflow

- [X] Ticket exists in Jira https://uktrade.atlassian.net/browse/TICKET_ID_HERE
- [X] Jira ticket has the correct status.
- [X] A clear/description pull request messaged added.

### Merging

- [x] This PR can be merged by reviewers. (If unticked, please leave for the author to merge)

### CODE REVIEW

File: contact/templates/domestic/contact/export-support/step-2.html [hunk 1]
```diff
@@ -6,6 +6,12 @@
 
   document.querySelectorAll('[data-hidden]').forEach(el => el.style.display = 'none');
 
+  let hiddenFields = document.querySelectorAll('[data-hidden]');
```
Comment (LuptonM) [inline contact/templates/domestic/contact/export-support/step-2.html:9]: const since we arent reassigning

File: contact/mixins.py [hunk 2]
```diff
@@ -65,6 +65,8 @@ class DomesticExportSupportStep2Mixin(forms.Form):
             ('50-249', '50 to 249'),
             ('250-499', '250 to 499'),
             ('500plus', 'More than 500'),
+            ('dontknow', "I don't know"),
+            ('prefernottosay', "I'd prefer not to say"),
```
Comment (mforner13) [inline contact/mixins.py:69]: The convention for choice values is normally snake case - although I see it's the same in bits of the form, so could do a separate PR to adjust them all :)

File: contact/templates/domestic/contact/export-support/confirmation.html [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+        <li>longer to reply to your enquiry</li>
+        <li>more information to answer your enquiry</li>
+    </ul>
+    <hr class="great-hr-blue great-hr-blue--thin" />
+
+    {% for field in form.visible_fields %}
+        {% if field.name == 'help_us_further' %}
+        <div class="govuk-form-group govuk-!-margin-bottom-8 {% if field.errors %}govuk-form-group--error{% endif %}">
+            {% include 'domestic/contact/includes/govuk-form-field.html' with field=field optional=True %}
+        </div>
```
Comment (mforner13) [inline contact/templates/domestic/contact/export-support/confirmation.html:24]: If the only difference for `help_us_further` is that optional=True, I think you should be able to just use field.required and have one loop like:
```
{% for field in form.visible_fields %}
<div class="govuk-form-group govuk-!-margin-bottom-8 {% if field.errors %}govuk-form-group--error{% endif %}">
{% include 'domestic/contact/includes/govuk-form-field.html' with field=field optional=field.required %}
</div>
{% endfor %}
```

**Review by mforner13 — APPROVED** [review]
